llvm/utils/TableGen/Basic/CodeGenIntrinsics.h

```diff
@@ -139,12 +139,12 @@ struct CodeGenIntrinsic {
 
   bool hasProperty(enum SDNP Prop) const { return Properties & (1 << Prop); }
 
-  /// Goes through all IntrProperties that have IsDefault
-  /// value set and sets the property.
-  void setDefaultProperties(Record *R, ArrayRef<Record *> DefaultProperties);
+  /// Goes through all IntrProperties that have IsDefault value set and sets
+  /// the property.
+  void setDefaultProperties(ArrayRef<const Record *> DefaultProperties);
 
   /// Helper function to set property \p Name to true;
```

Comment:
Since you're here. Can you change the semicolon at the end of this comment to a period.

Reply:
Will do.

Reply:
Done.